### PR TITLE
Cql4bonnie remove qdm

### PR DESCRIPTION
**Pull request by mayerm94**
Bonnie_bundler had failing tests because of an error in a backend HQMF execution engine that was only used by the bundle feature. That error spawned from an upgrade in mongodb from 2.* to > 3.0. Since we do not use bundles with CQL, it is safe to remove the functionality and the tests that have been causing failed travis builds. This PR removes the bundle feature and HQMF loading. Should be tested along side projecttacoma/bonnie#860

Pull requests into Bonnie Bundler require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-1117
- [x] JIRA ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced. N/A
- [x] Tests are included and test edge cases N/A Tests modified for changes
- [x] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [x] Automated regression test(s) pass

**Reviewer 1:** @jbradl11 

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree

**Reviewer 2:** @aelliott

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree

### CODE REVIEW

**Comment by mayerm94** [comment]
Bonnie_bundler had failing tests because of an error in a backend HQMF execution engine that was only used by the bundle feature. That error spawned from an upgrade in mongodb from 2.* to > 3.0. Since we do not use bundles with CQL, it is safe to remove the functionality and the tests that have been causing failed travis builds. This PR removes the bundle feature and HQMF loading. Should be tested along side https://github.com/projecttacoma/bonnie/pull/860

https://jira.mitre.org/browse/BONNIE-1117

File: lib/measures/loading/mat_loader.rb [hunk 1]
```diff
@@ -11,14 +11,12 @@ def self.load(file, user, measure_details, vsac_user=nil, vsac_password=nil, ove
     end
 
     def self.mat_export?(zip_file)
-      Measures::CqlLoader.mat_cql_export?(zip_file) || Measures::QDMLoader.mat_hqmf_export?(zip_file)
```
Comment (jbradl11) [inline lib/measures/loading/mat_loader.rb]: This file was used as an abstraction layer for uploading QDM and CQL based measures. Now that we are not supporting QDM+HQMF measures, this file could be removed and we could just use the CqlLoader class. This may be outside of the scope of this PR.

File: lib/measures/loading/value_set_loader.rb [hunk 2]
```diff
@@ -1,120 +1,24 @@
...
+        vsm.save!
       end
     end
 
-
     def self.get_value_set_models(value_set_oids, user=nil)
       HealthDataStandards::SVS::ValueSet.by_user(user).in(oid: value_set_oids)
     end
 
-    def self.load_value_sets_from_xls(value_set_path)
```
Comment (alexanderelliott121) [inline lib/measures/loading/value_set_loader.rb:23]: This still looks like it is in use in one rake task
Parser_comparison.rake

File: lib/measures/loading/sources_loader.rb [hunk 3]
```diff
@@ -1,75 +0,0 @@
-module Measures
-  
-  # Utility class for loading measure definitions into the database from source files
-  class SourcesLoader
```
Comment (alexanderelliott121) [inline lib/measures/loading/sources_loader.rb:4]: Used 
bonnie/lib/tasks/dashboard.rake
and 
bonnie/lib/tasks/sources.rake

File: lib/measures/loading/value_set_loader.rb [hunk 4]
```diff
@@ -1,120 +1,24 @@
...
-        if match
-          vs.concepts = concepts
-          vs.save!
-        end
-      end
-      puts "deleted #{white_delete_count} white / #{black_delete_count} black list entries"
-    end
-
-    def self.load_white_list(white_list_path, user =nil)
-      parser = HQMF::ValueSet::Parser.new()
```
Comment (alexanderelliott121) [inline lib/measures/loading/value_set_loader.rb:54]: Also used in bonnie rake task

File: lib/measures/loading/value_set_loader.rb [hunk 5]
```diff
@@ -1,120 +1,24 @@
...
-        existing.concepts = concepts
-        existing.save!
-
-      end
-      puts "loaded: #{white_list_total} white list entries"
-
-    end
-
-    def self.load_black_list(black_list_path, user = nil)
-      parser = HQMF::BlackList::Parser.new()
```
Comment (alexanderelliott121) [inline lib/measures/loading/value_set_loader.rb:91]: Used in bonnie rake task

**Review by alexanderelliott121 — CHANGES_REQUESTED** [review]

File: lib/models/cql_measure.rb [hunk 6]
```diff
@@ -148,4 +148,9 @@ def count_expression_logical_branches(expression)
     self.complexity
   end
 
+  # Compute a simplified diff hash for Complexity Dashboard usage; stored within measure.latest_diff
+  def diff(other)
+    HQMF::Measure::LogicExtractor.get_measure_logic_diff(self,other,true)
```
Comment (jbradl11) [inline lib/models/cql_measure.rb]: I don't think LogicExtractor will work as is with CQL measures. Has this been tested?

File: test/unit/measure_complexity_test.rb [hunk 7]
```diff
@@ -1,18 +0,0 @@
...
-class MeasureComplexityTest < ActiveSupport::TestCase
-
-  setup do
-    dump_db
-    bundle = File.new File.join('test','fixtures','bundle-tiny.zip')
-    Measures::BundleLoader.load(bundle, nil)
-  end
-
-  # Just a regression test
-  test "calculate complexity" do
```
Comment (jbradl11) [inline test/unit/measure_complexity_test.rb]: I believe we still calculate complexity on CQL measures. I think we should have a test for it. Might be out of scope for this PR?

**Review by alexanderelliott121 — APPROVED** [review]
Referring rake tasks cleaned up

File: test/unit/load_measure_logic_test.rb [hunk 8]
```diff
@@ -1,20 +0,0 @@
-require 'test_helper'
-
-class LoadMeasureLogicTest < ActiveSupport::TestCase
-
-  setup do
-    @mat_export = File.new File.join('test','fixtures','07_ExclusiveBreastMilkFeeding_Artifacts.zip')
-  end
-
-  test "Loading a MAT export zip file" do
```
Comment (jbradl11) [inline test/unit/load_measure_logic_test.rb:9]: Do we have something similar to this for CQL measures?

Comment (mayerm94) [reply test/unit/load_measure_logic_test.rb:9]: test/unit/measure_diff_test.rb

**Review by jbradl11 — DISMISSED** [review]
